Source/Tools/TestHarnessRunner/TestHarnessRunner.cpp

```diff
@@ -198,13 +198,25 @@ int main(int argc, char **argv, char **const envp) {
   FEXCore::Config::Load();
 
   auto Args = FEX::ArgLoader::Get();
+  auto Filename = Args[0];
+  auto ConfigFile = Args[1];
 
   if (Args.size() < 2) {
     LogMan::Msg::EFmt("Not enough arguments");
     return -1;
   }
 
-  FEX::HarnessHelper::HarnessCodeLoader Loader{Args[0], Args[1]};
+  if (!std::filesystem::exists(Filename)) {
```

Comment:
These cause the glibc fault CI test to fail because of these concerns here: https://github.com/FEX-Emu/FEX/blob/main/docs/ProgrammingConcerns.md#stdfilesystemexists

Easy enough switch will be to switch both use cases over to `FHU::Filesystem::Exists`

Reply:
TIL - following your suggestion.